test/functional/feature_taproot.py

```diff
@@ -1181,15 +1181,11 @@ def spenders_taproot_inactive():
     ]
     tap = taproot_construct(pub, scripts)
 
-    # Test that keypath spending is valid & non-standard, regardless of validity.
+    # Test that valid spending is standard.
     add_spender(spenders, "inactive/keypath_valid", key=sec, tap=tap, standard=Standard.V23)
-    add_spender(spenders, "inactive/keypath_invalidsig", key=sec, tap=tap, standard=False, sighash=bitflipper(default_sighash))
-    add_spender(spenders, "inactive/keypath_empty", key=sec, tap=tap, standard=False, witness=[])
-
-    # Same for scriptpath spending (and features like annex, leaf versions, or OP_SUCCESS don't change this)
     add_spender(spenders, "inactive/scriptpath_valid", key=sec, tap=tap, leaf="pk", standard=Standard.V23, inputs=[getter("sign")])
-    add_spender(spenders, "inactive/scriptpath_invalidsig", key=sec, tap=tap, leaf="pk", standard=False, inputs=[getter("sign")], sighash=bitflipper(default_sighash))
-    add_spender(spenders, "inactive/scriptpath_invalidcb", key=sec, tap=tap, leaf="pk", standard=False, inputs=[getter("sign")], controlblock=bitflipper(default_controlblock))
+
+    # Test that features like annex, leaf versions, or OP_SUCCESS are valid but non-standard
```

Comment:
Is there any value to keeping any of the `inactive/` test cases? They aren't being tested with taproot inactive anymore, and the behaviour when taproot is active should already be being tested elsewhere...

Reply:
Good point. Currently they are used to run against previous releases, but given that the tests passed once one the previous releases (and previous releases are "constant"), we can probably remove this code. Happy to look into this as a follow-up unless reviewers want me to do it here.

Reply:
I'm confused about what the test is doing with respect to previous releases, and wonder if the name `spenders_taproot_inactive` is still appropriate.

When run with `--previous_release, it runs against v0.20.1 which does not have Taproot, so that makes sense. But otherwise it sets `-vbparams=taproot:1:1`, which is a no-op as of this PR.

Reply:
I'll rename the function name with the other test cleanups in a follow-up. I'd say we should focus on the consensus change here and leave style changes in the test for later.

Reply:
Renamed the function (among other cleanup) in https://github.com/bitcoin/bitcoin/pull/24737